### ISSUE TITLE
🐛 Biome JSON schema adds unnecessary angle brackets to rule hyperlinks, confusing IDE link detection

### ISSUE THREAD

**Issue by Bertie690** [biomejs/biome#8761]
### Environment information

<details>

```bash

CLI:
  Version:                      2.3.11
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           linux

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  BIOME_THREADS:                unset
  NO_COLOR:                     unset
  TERM:                         xterm-256color
  JS_RUNTIME_VERSION:           v24.5.0
  JS_RUNTIME_NAME:              node
  NODE_PACKAGE_MANAGER:         pnpm/10.28.0

Biome Configuration:
  Status:                       Loaded successfully
  Path:                         biome.jsonc
  Formatter enabled:            true
  Linter enabled:               true
  Assist enabled:               true
  VCS enabled:                  true
  HTML full support enabled:    unset

Workspace:
  Open Documents:               0


```

</details>

### What happened?

1. Set a rule to "off" inside a config file
2. Hover over the rule in an IDE
3. Realize the link has extra angle brackets and is thus invalid

<img width="556" height="98" alt="Image" src="https://github.com/user-attachments/assets/3821f3af-bf54-4b10-a1c9-50a552a5f1b3" />

### Expected result

It should not incldue angle brackets, which throw off IDE link detection and lead to invalid hyperlinks.
(VS Code took me to https://biomejs.dev/linter/rules/no-document-cookie/>)

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by ematipico** [comment]
cc @Netail

**Comment by Netail** [comment]
Oh huh, the `<>` weren't there in the original PR, seems to be added in https://github.com/biomejs/biome/pull/8137. Not sure why those were added @denbezrukov 

Both scenarios (With `<>` & without `<>`) do not seem to work for IntelliJ, in case that was the reasoning. But, for vscode with `<>` works, without `<>` breaks the url. I guess we can revert it to no `<>` as it won't make a difference for IntelliJ, but does fix for VSCode.

<table>
<tr>
 <td>
 <td>VSCode
 <td>IntelliJ
<tr>
 <td>Without `<>`
 <td><img width="451" height="81" alt="Image" src="https://github.com/user-attachments/assets/bcfa92ce-6411-42b9-ab19-3be9fae065eb" />
 <td><img width="515" height="203" alt="Image" src="https://github.com/user-attachments/assets/19c12969-710f-4fa2-a1e6-8343f4f84be2" />
<tr>
 <td>With `<>`
 <td><img width="419" height="78" alt="Image" src="https://github.com/user-attachments/assets/b616114d-e6b2-49fb-a15e-d63651053e65" />
 <td><img width="378" height="198" alt="Image" src="https://github.com/user-attachments/assets/fa339c5d-3da6-4107-8eef-b23ac0f4be61" />
</table>

**Comment by denbezrukov** [comment]
Oh, wow, I'm sorry - it seems these changes accidentally slipped in from another commit(

**Comment by Netail** [comment]
> Oh, wow, I'm sorry - it seems these changes accidentally slipped in from another commit(

No worries, will create a revert. Making sure there wasn't a specific reason :)

**Comment by Netail** [comment]
Too bad IntelliJ doesn't make them clickable, real bummer 

- https://youtrack.jetbrains.com/issue/IJPL-161269/Make-URLs-in-Quick-documentation-coming-from-JSON-Schema-descriptions-clickable
- https://youtrack.jetbrains.com/issue/IJPL-161267/Newline-characters-from-JSON-Schema-descriptions-are-not-rendered-properly-in-Quick-documentation-action
